hack/deploy-helm.sh

```diff
@@ -1,6 +1,5 @@
 #!/usr/bin/env bash
 # Copyright 2024 The HAMi Authors.
-#
```

Comment:
We try to keep the status quo for headers, unless there is a special reason to modify them